history-of-quakers/modernized/13-chapter-11.adoc

```diff
@@ -412,12 +449,16 @@ that they had no warrant from the word of God, nor precedent from our country,
 nor power from his majesty to hang the man.
 I rest,
 
+[.signed-section-closing]
 Your Friend,
 
-THOMAS WILKIE.
+[.signed-section-signature]
+Thomas Wilkie.
 
 To Mr. George Lad, master of the "`America`" of Dartmouth, now at Barbados.
```

Comment:
Here is another "To...." at the end of the letter instead of the beginning. Not sure what to do with those. I just realized that I think I missed that you sent the last PR back to me that probably has the answer to this there. Sorry!! 😄